backend/app/services/chat/task_default_knowledge_bases.py

```
@@ -75,38 +89,237 @@ def _iter_team_member_default_knowledge_base_ids(
+    user: User,
+    ref: DefaultContextRef,
+    bound_at: str,
+) -> tuple[dict[str, Any] | None, dict[str, Any] | None]:
+    if ref.type != "dingtalk_doc":
+        return None, None
+
+    if not _is_dingtalk_mcp_configured(user, ref.source):
+        return None, _build_context_warning(
+            ref,
+            "mcp_not_configured",
+            "未开启钉钉 MCP，无法读取钉钉知识",
```

Ruff (0.15.17)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)